README.md

```diff
@@ -44,25 +44,25 @@ Build the library.
 
 ```javascript
 // Import the Subspace.js library.
-import { SubspaceClient, Identity } from "subspace.js";
+import { SubspaceClient, Identity } from 'subspace.js'
 
 // Generate an Identity from node.js
-const identity = await Identity.fromUri(mnemonic);
+const identity = await Identity.fromUri(mnemonic)
```

Comment:
Why deleting semicolons everywhere?

Reply:
I'm this was resolved without any fixes and comments, un-resolving then

Reply:
Sorry i will re create my responses, cancelled my own review losing all comments.

As response to this, i was just trying to make it simpler, like less characters for the eyes. think it looks better. I can rollback to the previous format.

Reply:
Not sure how it is easier to read :confused:
Personally I automatically look for the remaining of the code unless I see semicolon, so the benefit is questionable at best, I'd leave it as it was instead.

Reply:
Sure, i made the changes ;)